### ISSUE TITLE
Opening a document in notepad++ via command line will discard unsaved changes.

### ISSUE THREAD

**Issue by jamesford42** [notepad-plus-plus/notepad-plus-plus#1470]
Situation is...

Working in visual studio, solution contains some .cfg files.
Notepad++ is set as the 'default editor' for .cfg files (this is a visual studio level file association not a windows level file association).
Double clicking or choosing 'open' launches notepad++ and opens the document.
Make some changes to document in notepad++ but don't save it.
Double clicking or choosing 'open' on that same file in visual studio results in the unsaved changes being lost. The document is reloaded from disk.

I assume that visual studio is just launching notepad++ with some command line arguments like 'open [path]'. So most likely there are other ways of reproducing this bug and it is, most likely, nothing to do with visual studio.

**Comment by dail8859** [comment]
Could you include your debug info (from the menu `? > Debug Info`). 

Also, what settings are you using under `Settings > Preferences > Backup`

**Comment by filipbartek** [comment]
I reproduced the steps locally and did not observe the reported bad behavior. Details follow.

I made Notepad++ the default program to open .cfg files by right-clicking a .cfg file in Solution Explorer, choosing Notepad++ and clicking "Set as Default". Having done that, when I opened the file from Solution Explorer, made some unsaved changes and opened the file from Solution Explorer again, the unsaved changes remained intact in Notepad++.

Configuration:
* Notepad++: 7.5.6 64-bit
    * Preferences / Backup:
        * Remember current session for next launch: Disabled
        * Backup on save: None
* Visual Studio: 2015 (14.0.25431.01 Update 3)
